src/carbonio-files-ui-common/utils/utils.ts

```diff
@@ -802,7 +841,10 @@ export const getListItemAvatarPictureUrl = (
 	version: number | undefined,
 	type: NodeType,
 	mimeType: string | undefined
-): string | undefined => getPreviewThumbnailSrc(id, version, type, mimeType, 80, 80);
+): string | undefined =>
+	includes(mimeType, 'pdf')
+		? undefined
+		: getPreviewThumbnailSrc(id, version, type, mimeType, 80, 80);
```

Comment:
I think it's better if clean a bit the places where we check if the preview/thumbnail is supported to only have 1 point in which we do the conditional.
This point could be the funciton `isSupportedByPreview`, which can be re-organized with a if/then shape instead of the logical operators (since the conditionals are beginning to be complex).
We could use the objects `thumbnailHandledMimeTypes` and `previewHandledMimeTypes` to check also for the pdf mimetype support, and then inside `getPreviewThumbnailSrc` we could use `isSupportedByPreview` to retrieve both the support check and the preview/thumbnail type based on the mimeType